packages/forma-36-website/src/content/migration-v3-to-v4.mdx

```diff
@@ -46,6 +46,8 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 - **Modal**: We changed the way you import the Modal component. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-modal-components).
 
+- **Tooltip**: Two props got renamed: `containerElement` to `as`; and `place` to `placement`. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tooltip-components).
```

Comment:
We should remove the semicolon before **and**. Other than that, it looks great.

Reply:
Cool! Thanks a lot, I will do the changes